app/service/file_svc.py

```diff
@@ -78,12 +81,16 @@ def __init__(self, plugins, exfil_dir):
         return path
 
     async def _compile(self, name, platform):
+        abilities = await self.data_svc.explode_abilities()
+        platforms = {ab['platform'].lower() for ab in abilities}
```

Comment:
let's move these two lines (84,85) to the __init__ function, as they won't change after the server boots up (and we can avoid re-initializing them on each file download. so in the init, it would be:
```
self.valid_platforms = {ab['platform'].lower() for ab in await self.data_svc.explode_abilities()}
```

Reply:
I add "async def init()" in file_svc to load platforms and call it from init function of server.py after attach_plugins.
Is it correct operation to load data from init?